src/picosvg/svg.py

```diff
@@ -39,6 +39,7 @@
     xlinkns,
     parse_css_declarations,
     parse_view_box,
+    _LinkedDefault,
```

Comment:
might as well make it linked_default?

Reply:
it's a class, usually in python you use TitleCase for those

Reply:
Oh it's a class, then just remove the _?

Reply:
ok.. it's just that it's kind of a hack so i'd rather mark it as private (`_` prefixed)

Reply:
Your very first use is importing it somewhere else so ... it's not *really* private